build.gradle

```diff
@@ -61,6 +61,7 @@ dependencies {
     compile 'org.codehaus.sonar.runner:sonar-runner-api:2.4'
     compile 'org.codehaus.sonar-plugins:sonar-issues-report-plugin:1.3'
     compile 'com.jayway.jsonpath:json-path:0.9.1'
+    compile 'org.eclipse.jgit:org.eclipse.jgit:4.1.0.201509280440-r'
```

Comment:
We use Gerrit REST API to communicate. This dependency is a second way to access git. I'd like to avoid that and not add this dependency at all.